skills/dpdata-cli/SKILL.md

```
@@ -0,0 +1,178 @@
+
+```bash
+uvx dpdata OUTCAR -i vasp/outcar -O data.h5 -o deepmd/hdf5
+```
+
+## Supported Formats
+
+Formats may be updated. For the complete and latest list, see:
+- [Formats Reference (stable)](https://docs.deepmodeling.com/projects/dpdata/en/stable/formats.html)
+
+### DeePMD-kit Formats
+| Format Name | Description |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)